MonumentAddons.cs

```diff
@@ -3082,11 +3058,11 @@ private List<BaseMonument> GetMonumentsByAliasOrShortName(string aliasOrShortNam
                 return cargoShipList.Count > 0 ? cargoShipList : null;
             }
 
-            var monuments = FindMonumentsByAlias(aliasOrShortName);
+            var monuments = _pluginInstance._monumentHelper.FindMonumentsByAlias(aliasOrShortName);
             if (monuments.Count > 0)
                 return monuments;
 
-            return FindMonumentsByShortName(aliasOrShortName);
+            return _pluginInstance._monumentHelper.FindMonumentsByShortName(aliasOrShortName);
```

Comment:
Can remove `_pluginInstance.` since this is already inside an instance method of the plugin class.